src/pages/events/[eventId]/edit/index.page.tsx

```diff
@@ -0,0 +1,7 @@
+import { getApplicationServerSideProps } from '@/utils/getApplicationServerSideProps';
+
+import { EventForm } from '../../../create/EventForm';
```

Comment:
Not sure about the approach here.
Do use the existing EventForm from `create` or do we create a new `EventForm` inside the `edit` folder? 
I feel like they are going to use a lot of the same functionalities?

Reply:
I think this way is good